packages/x-components/src/design-system/components/picture/default.scss

```diff
@@ -1,48 +1,20 @@
 .x-picture {
-  // layout
-  display: block;
-  overflow: hidden;
-
-  // border
-  border-radius: var(--x-size-border-radius-top-picture-default)
-    var(--x-size-border-radius-right-picture-default)
-    var(--x-size-border-radius-bottom-picture-default)
-    var(--x-size-border-radius-left-picture-default);
-
-  // color
-  background: var(--x-color-background-picture-default);
-
   &__image {
-    // layout
-    display: block;
-
```

Comment:
Idem, as this class ends up being applied on `div`, `img` and `svg` and all of them have `display: block` applied by default in browsers.